### PR TITLE
Add direct insert support

### PR DESCRIPTION
**Pull request by nicosuave**

### CODE REVIEW

**Comment by nicosuave** [comment]
I might be missing something on the tests; don't write very much Go; would appreciate guidance

**Comment by nicosuave** [comment]
Also realizing my approach is naive after more closely reading https://github.com/slingdata-io/sling-cli/issues/35

**Comment by yokofly** [comment]
I just saw this pr, sorry, last week I was on holiday, so there was some delay.
Can the current impl solve the OOM mentioned in the DC? https://discord.com/channels/1210406123744796702/1210406123744796705/threads/1291805329025269791

the old issue needs to do 2 things:
one for refactor
another for the direct insert.
But if the current direct insert works I think it is OK?

File: cmd/sling/tests/suite.db.template.tsv [hunk 1]
```diff
@@ -3,18 +3,18 @@ n	test_name	source_conn	source_stream	source_options	stream_config	target_conn	t
...
 3	discover_table					[conn]	[schema].[table]	discover				"{""validation_contains"": ""create_dt"", ""validation_row_count"": ""11"", ""level"": ""column""}"
 4	parquet_snapshot	local	file://tests/files/test1.parquet	{}		[conn]	[schema].[table]_snapshot	snapshot			"{""post_sql"" : ""drop table [schema].[table]_2""}"	"{""validation_row_count"": "">999"", ""validation_types"": {""code"":""decimal"",""create_dt"":""timestamp"",""date"":""timestamp"",""rating"":""decimal"",""target"":""bool""}}"
-5	parquet_truncate	local	file://tests/files/test1.parquet	"{""columns"": {""rating"": ""float""}}"		[conn]	[schema].[table]_truncate	truncate			"{""pre_sql"": ""drop table [schema].[table]_snapshot "", ""post_sql"" : ""drop table [schema].[table]_truncate""}"	
+5	parquet_truncate	local	file://tests/files/test1.parquet	"{""columns"": {""rating"": ""float""}}"		[conn]	[schema].[table]_truncate	truncate			"{""pre_sql"": ""drop table [schema].[table]_snapshot "", ""post_sql"" : ""drop table [schema].[table]_truncate""}"
```
Comment (yokofly) [inline cmd/sling/tests/suite.db.template.tsv:6]: image
image

last time I struggled with this display as well, is there any reason we change the blank?
https://github.com/slingdata-io/sling-cli/blob/5e59b15b6c92d68f2b091118951aa6bafd7be6f6/cmd/sling/tests/suite.db.template.tsv

https://github.com/slingdata-io/sling-cli/blob/1.2.21/cmd/sling/tests/suite.db.template.tsv

Comment (nicosuave) [reply cmd/sling/tests/suite.db.template.tsv:6]: Ah my editor did some autoformat; thanks!

**Comment by nicosuave** [comment]
> Can the current impl solve the OOM mentioned in the DC? https://discord.com/channels/1210406123744796702/1210406123744796705/threads/1291805329025269791

I haven't explicitly built & tested, but that was me reporting the issue

**Comment by flarco** [comment]
@nicosuave thank you for spending time of this and opening a PR. This function needs a good refactor for it to be laid out clearly and work properly, as it is a bit of a mess. Since @yokofly is working on this (https://github.com/slingdata-io/sling-cli/pull/399), I'm going to use his PR as the base for these changes. Goal is to have this for the 1.2.21 release.
Closing this.

**Pull request closed without merging**